### PR TITLE
PAYARA-3038/3077 Jersey JAXB Unmarshalling Error

### PR DESCRIPTION
**Pull request by jGauravGupta**
Fixes : https://github.com/payara/Payara/issues/3077

Signed-off-by: Gaurav Gupta <gaurav.gupta.jc@gmail.com>

### CODE REVIEW

File: core-server/src/main/java/org/glassfish/jersey/server/internal/inject/ParamConverterFactory.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2012, 2018 Oracle and/or its affiliates. All rights reserved.
+ * (Portions) Copyright (c) 2018 Payara Foundation and/or its affiliates.
```
Comment (jansupol) [inline core-server/src/main/java/org/glassfish/jersey/server/internal/inject/ParamConverterFactory.java]: No (Portions) please

Comment (mkarg) [reply core-server/src/main/java/org/glassfish/jersey/server/internal/inject/ParamConverterFactory.java]: What is the legal implication of such a "Portions" line?

Comment (jansupol) [reply core-server/src/main/java/org/glassfish/jersey/server/internal/inject/ParamConverterFactory.java]: The "(Portions)" word just makes the Glassfish Copyright plugin check fail. Multiple Copyright lines passes. We have discussed this with the previous Payara PR.

**Comment by jansupol** [comment]
Closing in favor of #3955

**Pull request closed without merging**